modules/jupyter-widget/src/create-deck.js

```diff
@@ -41,8 +42,13 @@ const jsonConverter = new deck.JSONConverter({
   configuration: jsonConverterConfiguration
 });
 
-export function updateDeck(inputJSON, deckgl) {
-  const results = jsonConverter.convert(inputJSON);
+export function updateClasses({className, resourceUri, onComplete}) {
```

Comment:
Why do we need a function that directly calls another? Where is `onComplete` used?

Reply:
onComplete is used in the test associated with this function